hawc/apps/assessment/models.py

```diff
@@ -257,6 +257,10 @@ def get_rob_name_default():
     )
     created = models.DateTimeField(auto_now_add=True)
     last_updated = models.DateTimeField(auto_now=True)
+    epi_version = models.PositiveSmallIntegerField(
```

Comment:
add an `IntegerChoices` in the `assessment.constants` instead and revise the help-text

Reply:
and also, please keep the created and last_updated fields at the end and move this field above them